pkg/oc/cli/image/append/append.go

```diff
@@ -212,9 +213,9 @@ func (o *AppendImageOptions) Run() error {
 		return err
 	}
 	creds := dockercredentials.NewLocal()
-	ctx := context.Background()
+	ctx := apirequest.NewContext()
```

Comment:
Why this context change?

Reply:
Wanted parity with [the way image/mirror does it](https://github.com/openshift/origin/blob/master/pkg/oc/cli/image/mirror/mirror.go#L239).